CI/Bitucket/README.md

```diff
@@ -0,0 +1,73 @@
+# Requirements
+
+* Repository Bitbucket
+
+ ![Create repository](images/create-repository.png)
+ 
+* Application configured to deploy with serverless brefphp
+
+# Step by step
+
+## Bitbucket Configuration
+
+* Go to the "Settings -> Pipelines -> Settings" menu and enable Pipeline:
+![Enable Pipeline](images/enable-pipeline.png)
+ 
+* Access the menu "Settings -> Pipelines -> Deployments"
+    - Select the environment you want to deploy (Staging or Production), in this example I will use Staging
+    - Add the variables
+        * AWS_CREDENTIAL_KEY: Access Key ID
+        * AWS_CREDENTIAL_SECRET: Secret Key
+  
+![Set Environments](images/environment-variables.png)
+
+## Project configuration
+
+* In the root directory of your project create the file `bitbucket-pipelines.yml`:
+```
+pipelines:
+  branches:
+    master: # Whenever there is a push on the master branch
+      - step: # Step 1: Using a docker php image to install composer and download your project dependencies
+          name: Composer install
+          image: php:7.3
+          caches:
+            - composer
+          script:
+            - apt-get update && apt-get install -y unzip
+            - curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+            - composer install --prefer-dist --optimize-autoloader --no-dev
```

Comment:
Maybe here you could add a comment to explain that users can add any extra step they want, for example building the Symfony or Laravel cache (which will be a common use case)?

Reply:
Ok